src/cvi/modules/XB.py

```diff
@@ -0,0 +1,166 @@
+"""
+The Xie-Beni (XB) Cluster Validity Index.
+"""
+
+# Custom imports
+import numpy as np
+
+# Local imports
+from . import _base
+
+
+# XB object definition
+class XB(_base.CVI):
+    """
+    The stateful information of the Xie-Beni (XB) Cluster Validity Index.
+
+    References
+    ----------
+    1. X. L. Xie and G. Beni, "A Validity Measure for Fuzzy Clustering," IEEE Transactions on Pattern Analysis and Machine Intelligence, vol. 13, no. 8, pp. 841-847, 1991.
+    2. M. Moshtaghi, J. C. Bezdek, S. M. Erfani, C. Leckie, and J. Bailey, "Online Cluster Validity Indices for Streaming Data," ArXiv e-prints, 2018, arXiv:1801.02937v1 [stat.ML]. [Online].
+    3. M. Moshtaghi, J. C. Bezdek, S. M. Erfani, C. Leckie, J. Bailey, "Online cluster validity indices for performance monitoring of streaming data clustering," Int. J. Intell. Syst., pp. 1-23, 2018.
+    """
+
+    def __init__(self):
+        """
+        XB initialization routine.
+        """
+
+        # Run the base initialization
+        super().__init__()
+
+        # XB-specific initialization
```

Comment:
make these private with "_" prefix

Reply:
I think that what you highlighted might be off since lines 29-32 are comments and a super(), but I get your suggestion. Making a new PR since all CVIs are affected.

Reply:
#51